test/presenters/hotwire_combobox/component_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class HotwireCombobox::ComponentTest < ApplicationViewTestCase
+  setup do
+    @view = ActionView::Base.new(ActionController::Base.view_paths, {}, ActionController::Base.new)
+  end
+
+  test "hidden_field_attrs returns value from form object if available" do
+    existent_field_on_model = "Peter"
+    field_name = :name
```

Comment:
I think the test is clear enough that we don't need these explaining variables. Let's inline the values instead. Same goes for the other tests.

It's okay to introduce a bit of harmless repetition in order to keep less things in mind while reading these tests 👍